.travis.yml

```diff
@@ -16,6 +16,8 @@ stages:
   - name: unit-test
   # Builds are split into 'e2e-test' and 'build' to allow e2e tests to run first. If e2e fails, don't bother
   # building and pushing the images for the other architectures.
+  - name: minikube-e2e-test
+    if: branch = main AND fork = false AND type = cron
```

Comment:
The condition should be the same as build and build-manifest, so minikube tests are run when PR is merged into main (and the daily driver is updated):
`if: (branch = main OR tag =~ ^v) AND fork = false AND type != pull_request AND type != cron`